packages/data-grid/src/Grid.tsx

```diff
@@ -421,9 +421,18 @@ export const Grid = function Grid<T>(props: GridProps<T>) {
       if (s) {
         s.scrollLeft += deltaX;
         s.scrollTop += deltaY;
+        if (
+          !(
+            (
```

Comment:
would it be beneficial to add a comment here to detail what the next few lines check for - for future dev references